public/API/API_GetComments.php

```diff
@@ -84,8 +87,13 @@
         $query->whereNull('banned_at');
     })
     ->offset($offset)
-    ->limit($count)
-    ->get();
+    ->limit($count);
+
+if ($sortOrder == 1) {
+    $commentsQuery->orderBy('Submitted', 'DESC');
+}
```

Comment:
You'll probably want an explicit `orderBy` for ascending to. It's unsafe to assume that an unordered query returns sorted data.